src/User.php

```diff
@@ -75,6 +75,28 @@ public static function login(UserInterface $user): void {
     }
   }
 
+  /**
+   * Act on user unblock.
+   *
+   * @see civicrmote_user_update()
+   */
+  public static function unblock(UserInterface $user): void {
```

Comment:
I'd suggest to name it `onUnblock` as it doesn't unblock a user, but is called when a user is unblocked.